.circleci/config.yml

```diff
@@ -60,4 +61,27 @@ workflows:
             branches:
               only:
                 - main
+      - orb-tools/publish-dev:
+          name: publish_dev_orb
+          attach-workspace: false
+          checkout: true
+          orb-name: ministryofjustice/cla-end-to-end-tests
+          orb-path: orb.yml
+          publish-token-variable: CIRCLE_TOKEN
+          context:
+            - laa-cla-orbs-token
+      - orb-tools/increment:
+          name: publish_production_orb
+          segment: minor
```

Comment:
I was expecting `patch` here. I guess I don't have extremely strong feelings on it, but if every single change is automatically incrementing a minor version it seems odd to me. We're basically discarding the patch position.
I guess if we increment patch, we're effectively not gonna use both of the other positions as even if we manually increment sometimes it'll almost certainly only be one of the positions. It just seems odd, to me.

Reply:
Updated .circleci/config.yml to reflect comment